addons/point_of_sale/static/src/app/components/popups/number_popup/number_popup.xml

```diff
@@ -16,10 +16,10 @@
             <t t-set-slot="footer">
                 <div class="d-flex flex-grow-1 justify-content-center">
                     <button t-att-disabled="!props.isValid(state.buffer) and props.isValidBlocking" class="btn btn-primary btn-lg lh-lg o-default-button me-2" tabindex="1" t-on-click="confirm">
-                        <t t-esc="props.confirmButtonLabel || 'Ok'" />
+                        <t t-esc="confirmButtonLabel"/>
```

Comment:
Could also use a `t-if` and extra `<t t-else="">` node for the "Ok" instead of a getter 🤷